pkgs/development/r-modules/default.nix

```diff
@@ -1286,6 +1286,10 @@ let
       preConfigure = ''
         export JAVA_CPPFLAGS=-I${pkgs.jdk}/include/
         export JAVA_HOME=${pkgs.jdk}
+        substituteInPlace R/zzz.R.in \
+          --replace ".onLoad <- function(libname, pkgname) {" \
```

Comment:
probably better to use `--replace-fail` here

Reply:
thanks and done!